Doc/library/lzma.rst

```diff
@@ -324,6 +324,27 @@ Miscellaneous
 
 .. _filter-chain-specs:
```

Comment:
It should be right before `Specifying custom filter chains`.

Reply:
Thank you. Fixed now